index.html

```diff
@@ -24,10 +24,7 @@
     <section id="welcome-section">
       <h1>Hello! This is Raf&Ronny's project portfolio</h1>
       <p>We are software developers at Microverse..</p>
-    
-
-    
-      <h2 class="project-tile">Github Project</h2>
```

Comment:
To keep your HTML less repetitive and clean, I suggest you consider having fewer space between your code. This will make your HTML page easily readable by any programmer even newbies.